ecole-doctorale-de-sciences-po-author-date-french.csl

```diff
@@ -0,0 +1,440 @@
+      <text variable="URL" prefix="En ligne : " suffix=" "/>          
+    <text value="consulté le" prefix="[" suffix=" "/>
+       <date variable="accessed" form="text" suffix="">
+        <date-part name="day"/>
+            <date-part name="month"/>
+            <date-part name="year"/>
+          </date>
+      </if>
+    </choose>
+  </macro>
+  <macro name="title">
+    <choose>
+      <if type="bill book broadcast graphic legal_case legislation map manuscript motion_picture patent personal_communication report song" match="any">
+        <text variable="title" text-case="capitalize-first" font-style="italic"/>
+      </if>
+      <else-if type="article-journal article-newspaper article-magazine" match="any">
+        <group delimiter=", ">
+          <text variable="title" text-case="capitalize-first" quotes="true" font-style="normal"/>
+          <text variable="container-title" font-style="italic"/>
+        </group>
+      </else-if>
+ <else-if type="post-weblog" match="any">
+        <group delimiter=", ">
+          <text variable="title" text-case="capitalize-first" quotes="true" font-style="normal"/>
+          <text variable="container-title" font-style="italic" prefix="Blog "/>
+        </group>
+      </else-if>
+      <else-if type="thesis" match="any">
+        <group delimiter=", ">
+          <text variable="title" text-case="capitalize-first" font-style="italic"/>
+          <text variable="genre"/>
+        </group>
+      </else-if>
+      <else-if type="manuscript" match="any">
+        <group delimiter=",">
+          <text variable="title" text-case="capitalize-first" quotes="true" font-style="normal"/>
+          <text variable="genre" prefix=" "/>
+        </group>
+      </else-if>
+      <else-if type="chapter entry-dictionary entry-encyclopedia" match="any">
+        <group>
+          <group suffix=" ">
+            <text variable="title" text-case="capitalize-first" quotes="true"/>
+          </group>
+          <text macro="editor" prefix="in " suffix=". "/>
+          <text variable="container-title" text-case="capitalize-first" font-style="italic"/>
+        </group>
+      </else-if>
+      <else-if type="webpage" match="any">
+        <group>
+          <text variable="title" text-case="capitalize-first" font-style="italic" suffix=". "/>
+        </group>
+      </else-if>
+      <else-if type="article" match="any">
+      <group>
+          <text variable="title" text-case="capitalize-first" font-style="italic" suffix=". "/>
+        </group>
+      </else-if>
+      <else>
+        <text variable="title" quotes="true"/>
+      </else>
+    </choose>
+  </macro>
+<macro name="pub-place">
+    <choose>
+      <if type="bill book chapter entry-dictionary entry-encyclopedia thesis legal_case manuscript map paper-conference song" match="any">
+        <choose>
+          <if variable="publisher-place" match="any">
+            <text variable="publisher-place" suffix=" : "/>
```

Comment:
I think you want to set this as a delimiter between pub-place and publisher macro. Otherwise you can get [s.l.]Publisher, which seems pretty clearly wrong. Also, shouldn't the space before the colon not be a nbsp?

Reply:
@AnnaCouthuresIdrizi -- I don't understand why this is marked as resolved? If I'm wrong about the above, you'd need to say why. I'm very skeptical that the code as is produces correct output.

Reply:
Hello, 
I marked it as solved by mistake ; you're right.